.circleci/config.yml

```diff
@@ -1,5 +1,13 @@
 version: 2.1
 
+definitions:
+  - release_filters: &release-filters
```

Comment:
I keep forgetting how powerful yaml is. It's mostly for the better that I don't know.